explorer/src/Pos/Explorer/Socket/Methods.hs

```diff
@@ -302,6 +333,18 @@ notifyTxsSubscribers
 notifyTxsSubscribers cTxEntries =
     view csTxsSubscribers >>= broadcast @ctx TxsUpdated cTxEntries
 
+notifyEpochsLastPageSubscribers
+    :: forall ctx m . ExplorerMode ctx m
+    => EpochIndex -> ExplorerSockets m ()
+notifyEpochsLastPageSubscribers currentEpoch = do
+    recipients <- view $ csEpochsLastPageSubscribers
+    -- ^ subscriber
+    maxPage <- getEpochPagesOrThrow currentEpoch
+    -- ^ max no. of epoch pages
```

Comment:
`-- ^ max no. of epoch pages` to `-- ^ last epoch page` and change variable name to `lastPage`.
Otherwise, we can just call `epochPageSearch` with `Nothing` and get the first page.